receiver/k8sclusterreceiver/README.md

```diff
@@ -52,6 +52,23 @@ Example:
 The full list of settings exposed for this receiver are documented [here](./config.go)
 with detailed sample configurations [here](./testdata/config.yaml).
 
+### Feature Gate Configurations
+- `receiver.k8sclusterreceiver.reportCpuMetricsAsDouble`  (default = `false`): The k8s container and node cpu metrics 
+being reported by the k8sclusterreceiver are transitioning from being reported as integer millicpu units to being 
+reported as double cpu units to adhere to opentelemetry cpu metric specifications. Please update any monitoring this 
+might affect, the change will cause cpu metrics to be double instead of integer values as well as metric values will 
+be scaled down by 1000x. You can control whether the k8sclusterreceiver reports container and node cpu metrics in 
+double cpu units instead of integer millicpu units with the feature gate listed below. Note, feature gate identifiers 
+prefixed with - will disable the gate and prefixing with + or with no prefix will enable the gate.
+  - Start otelcol with feature gate enabled:
+    - otelcol {other_arguments} --feature-gates=receiver.k8sclusterreceiver.reportCpuMetricsAsDouble 
+  - Start otelcol with feature gate disabled:
+    - otelcol {other_arguments} --feature-gates=-receiver.k8sclusterreceiver.reportCpuMetricsAsDouble 
+  - For more details see:
+    - [collector.go where the the feature gate is registered](./internal/collection/collector.go)
+    - https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/8115
+    - https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/semantic_conventions/system-metrics.md#systemcpu---processor-metrics
+
```

Comment:
Please mention that the feature gate will be enabled by default let's say in 0.40.0.

Also please list all metric names that are affected in Otel format

Reply:
Updated to include the metric names. 
The current version of opentelemetry-collector-contrib is at v0.46.0. I put included some target versions for each stage of the feature toggle that are spaced out by 3 versions (v0.47.0, v0.50.0, v0.53.0). Let me know if you want different values for these.

Reply:
Sounds good to me